org/views.py

```diff
@@ -137,4 +141,47 @@ def Permission_Set_List(request,org_id):
             }
             response_object.append(value)
         return Response(response_object, status.HTTP_200_OK)
-    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
\ No newline at end of file
+    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN) 
+
+@swagger_auto_schema(
+    operation_id="get_group",
+    method='GET',
+    responses={
+        '200': set_example(responses.get_group),
+        '401': set_example(responses.user_unauthorized_401),
+        '404': set_example({"detail": "This organisation doesn't exist."}),
+        '400': set_example({"detail" : "You are not a member of this organisation"}),
+        '403': set_example({"detail": "You are not authorised to view this."}),
+    }
+)    
+@api_view(['get'])
+@permission_classes([IsAuthenticated])
+def GetGroup(request,org_id):
+    print(request.user)
+    try:
+        org = Org.objects.get(pk=org_id)
+        print(org)
+    except Org.DoesNotExist:
+        return Response({"detail":"This organisation doesn't exist."}, status.HTTP_404_NOT_FOUND)
+    try:
+        member = Member.objects.get(
+        user = request.user,
+        org = org
+        )
+    except Member.DoesNotExist:
+        return Response({"detail" : "You are not a member of this organisation"}, status.HTTP_400_BAD_REQUEST)
+       
+    # if member.permission_set.perm_obj.permissions[Permissions.IS_STAFF]:
+    if True:    
```

Comment:
`if True:` seriously? :rofl: :rofl: :rofl:

Reply:
sorry I forgot to remove that.. I was just checking that API 😅😅😅